modules/Components/src/client/ACASFormFields.coffee

```diff
@@ -397,7 +402,7 @@ class ACASFormLSCodeValueFieldController extends ACASFormAbstractFieldController
 				el: @$('.bv_editablePicklist')
 				collection: @pickList
 				selectedCode: mdl.get('value')
-				parameter: @options.modelKey
+				parameter: parameter
 				codeType: mdl.get 'codeType'
```

Comment:
(Part 1/3) This is part of the fix to an issue where the selected parameter in the PickList would display its id along with the string (ie. "solubilityc178" instead of "solubility")